vizro-core/docs/pages/user-guides/customizing-css.md

```diff
@@ -0,0 +1,330 @@
+[same as the page `title`](pages.md), but such a value might not be a valid CSS identifier.
+A suitable `id` must be unique across all models in the dashboard and should contain only alphanumeric
+characters, hyphens (`-`) and underscores (`_`). In particular, note that spaces are _not_ allowed.
+
+Suppose you want to hide the page title on one page only. Here's how you can achieve this:
+
+1. Give a valid `id` to the `Page`, for example `Page(id="page-with-hidden-title", title="Page with hidden title", ...)`.
+2. Identify the CSS class or CSS `id` you need to target. To hide the page title, you need to hide the container with the `id `right-header`.
+3. Use the `id` in combination with CSS selectors to change the relevant CSS properties.
+4. Add your custom css file to the `assets` folder as explained above.
+
+
+!!! example "Hide page title on selected pages"
+    === "my_css_file.css"
+    ```css
+    #page-with-hidden-title #right-header {
+      display: none;
+    }
+    ```
+    === "app.py"
+        ```py
+        import vizro.models as vm
+        from vizro import Vizro
+
+        page_one = vm.Page(
+            id="page-with-hidden-title",
+            title="Page with hidden title",
+            components=[vm.Card(text="""# Placeholder""")]
+        )
+
+        page_two = vm.Page(
+            title="Page with shown title",
+            components=[vm.Card(text="""# Placeholder""")]
+        )
+
+        dashboard = vm.Dashboard(pages=[page_one, page_two])
+        Vizro().build(dashboard).run()
+        ```
+
+        <img src=https://py.cafe/logo.png alt="py.cafe logo" width="30"><b><a target="_blank" href="https://py.cafe/vizro-official/vizro-multi-page-example">Run and edit this code in Py.Cafe</a></b>
+
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to add data to the data manager and parse YAML configuration
+        # See yaml_version example
+        pages:
+        - components:
+            - text: |
+                # Placeholder
+              type: card
+          title: Page with hidden title
+          id: page-with-hidden-title
+        - components:
+            - text: |
+                # Placeholder
+              type: card
+          title: Page with shown title
+        ```
+    === "Result"
+         [![PageTitle]][PageTitle]
+
+    [PageTitle]: ../../assets/user_guides/assets/css_page_title.png
+
+
+### Overwrite CSS for selected components
+To adjust CSS properties for specific components, like altering the appearance of a selected [`Card`][vizro.models.Card] rather than all `Card`s,
+you need to use a CSS selector that targets the right CSS property. If you're unfamiliar with CSS selectors,
+you can refer to this [tutorial](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selector_structure) for guidance.
+
+Let's say we want to change the background and font-color of a specific `Card`.
```

Comment:
```suggestion
Let's say we want to change the `background` and `font-color` of a specific `Card`.
```

Reply:
Or if you don't want to use code font, it'll be background and font color with no hyphen.